app/gui/src/dashboard/utilities/tailwindVariants.ts

```diff
@@ -22,10 +22,10 @@ export type ExtractFunction<T> =
 export type TVWithoutExtends<T> = ExtractFunction<T> & Omit<T, 'extend'>
 
 /** Props for a component that uses `tailwind-variants`. */
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export type VariantProps<T extends (...args: any) => any> = Omit<
-  TvVariantProps<T>,
-  'class' | 'className'
-> & {
-  variants?: ExtractFunction<T> | undefined
+// TODO: add support for styling individual slots
```

Comment:
```
// TODO[sg]: link to issue here
// Add support for styling individual slots.
```

Reply:
Added, though the format is a little bit different: 
` * TODO: @MrFlashAccount [add support for styling individual slots](https://github.com/enso-org/cloud-v2/issues/1643)`

Thoughs?

Reply:
sounds good to me. i think our current format is just a convention, so the above is fine for me

Reply:
although to be fair i don't think we technically need markdown. not that i'm _against_ it, but it does mean it's harder to split it onto two lines

Reply:
yeah, actually i use TODO highlight too. and was a bit sad that the current format makes the highlighting not happen (with default settings, at least)